pyperformance/_pyproject_toml.py

```diff
@@ -12,13 +12,18 @@
 
 import os.path
 import re
+import sys
 import urllib.parse
 
 import packaging.requirements
 import packaging.specifiers
 import packaging.utils
 import packaging.version
-import toml
+
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    import tomli as tomllib
```

Comment:
I prefer to use the ImportError strategy for this code.
We should consider alternative implementations and we can not guarantee the tomlib from those implementations.

Reply:
Thanks, done!